### ISSUE TITLE
No events received in "Events Manager", an error "Update SDK"

### ISSUE THREAD

**Issue by bakwc** [facebook/facebook-sdk-for-unity#603]
## Checklist

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Unity Editor Version: `2020.1.7f`
- Unity SDK Version: `11.0.0`
- Installation Platform & Verison: `iOS` any version

## Goals

I want to create ads, but it's impossible because no events and "update sdk error".

## Expected Results

Events successfully delivered.

## Actual Results

No events.

## Steps to Reproduce

Have an old sdk. Update it to a new one.

## Code Samples & Details

I already updated SDK to a latest version and released to app store twice. But still no events, and an "Update your apps to the latest Facebook SDK error".
How can I debug it? I already spent a week trying to integrate ads, but it's not working.

**Comment by bakwc** [comment]
Probably an issue is similar to #589 - there is no response or help there.
Why do you ignore your customers?

**Comment by owaisfb** [comment]
@bakwc We have the latest version in the [main branch](https://github.com/facebook/facebook-sdk-for-unity) (unity SDK v12 that picks up the corresponding v12.0 of native mobile SDKs) which should fix the issues that you are seeing.

We are aiming to cut a new branch of v12 sometime next week, sorry for the delay here!

**Comment by bakwc** [comment]
I will try to use a version from main branch and let you know.

**Comment by bakwc** [comment]
Your master branch is currently broken. I used a fix from - https://github.com/facebook/facebook-sdk-for-unity/pull/600, it helped. I will upload a new version to store and let you know if the issue is solved.

**Comment by bakwc** [comment]
I updated SDK and released app in store (spent another 3 days).
And still I have the same error:
<img width="359" alt="Снимок экрана 2022-04-11 в 19 54 24" src="https://user-images.githubusercontent.com/1684011/162781325-d44b2f93-831e-4a54-b718-b87af8be5ef2.png">

**Comment by bakwc** [comment]
Here is my Unity settings:
<img width="376" alt="Снимок экрана 2022-04-19 в 10 48 50" src="https://user-images.githubusercontent.com/1684011/163942919-8b157840-bd75-401a-bcff-4ed619b56cde.png">

It worked with the previous settings.

Here is a support answer:

`From what we are seeing the problem is that your app is not sending any events. Can you confirm if you have implemented sending app events in your app? Docs reference - https://developers.facebook.com/docs/app-events/unity/`

I installed plugin and entered my app ID. Auto-log sending is enable. How can I debug why there is no events?

**Comment by fahall** [comment]
Possibly related to: https://github.com/facebook/facebook-sdk-for-unity/issues/608

**Comment by bakwc** [comment]
Hey, why do you ignore your users? Is there no engineer who can sit and troubleshot it? I am a small indie developer, I'm trying to integrate it for already two months, and you just ping-pong me between your departments.

**Comment by fahall** [comment]
Bump. @JordiFB @owaisfb @iquintero

**Comment by dyunusemre** [comment]
Hey,  @bakwc you find any answer for this issue?

**Comment by bakwc** [comment]
No, currently trying to register a completely new app and try again.

**Comment by bakwc** [comment]
It is still not working. 6 months have passed.

**Comment by albert-pixel** [comment]
Same issue here, unable to send events :(

**Comment by bakwc** [comment]
Hey guys. Could you please help to troubleshoot this issue? If you don't know what's the reason - may be you can add additional logs?
Event manager does not see events. The problem is somewhere inside SDK.
Here is all the logs that I have after installing sdk: https://pastebin.com/nAQBQCD9
@JordiFB @owaisfb @iquintero
Could you please make an sdk branch with additional logging so I we can ensure that events are successfully sending from SDK side?

**Comment by xiaoxiangmuyu** [comment]
Same issue,no event reported.
UnityVersion:2020.3.35
FaceBook SDKVersion:v11.0
